### DISCUSSION TITLE
KBEHDT1

### DISCUSSION THREAD

**Discussion by AntiPontifex** [iptv-org/community#383] [Show and tell]
### Channel ID

KBEHDT1.us

### Stream URL

https://dacastmmd.mmdlive.lldns.net/dacastmmd/e478039df74e43898cb010c7b76f468b/chunklist_b4628000.m3u8?p=79&s=1675355665&e=1675355785&h=96d0561419428904f6cf9f10fbbbf720

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by BellezaEmporium** [comment]
https://dacastmmd.mmdlive.lldns.net/dacastmmd/e478039df74e43898cb010c7b76f468b/master.m3u8

**Comment by AntiPontifex** [comment]
Thanks, Lane.

**Comment by freearhey** [comment]
https://dacastmmd.mmdlive.lldns.net/dacastmmd/e478039df74e43898cb010c7b76f468b/chunklist_b4628000.m3u8?p=79&s=1675355665&e=1675355785&h=96d0561419428904f6cf9f10fbbbf720 (offline)
<img width="833" alt="image" src="https://user-images.githubusercontent.com/7253922/232255787-1e7e312a-5977-454f-a23f-d76aa6afae30.png">

https://dacastmmd.mmdlive.lldns.net/dacastmmd/e478039df74e43898cb010c7b76f468b/master.m3u8 (offline)
<img width="711" alt="image" src="https://user-images.githubusercontent.com/7253922/232255796-0d770e44-aa07-4648-bd76-46733c59e152.png">
